### ISSUE TITLE
Test error in 0.16.x after merging

### ISSUE THREAD

**Issue by benjaoming** [learningequality/ka-lite#4956]
## Summary

After merging #4942 there's a test failure... don't know if it's random or permanent

## Branch or installer method

 - Branch: 0.16.x

## Traceback or relevant snippet from server.log

https://circleci.com/gh/learningequality/ka-lite/2045?utm_campaign=build-failed&utm_medium=email&utm_source=notification

```

FAIL: test_playlist_progress (kalite.coachreports.tests.coachreport_tests.PlaylistProgressResourceTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/ubuntu/ka-lite/kalite/coachreports/tests/coachreport_tests.py", line 141, in test_playlist_progress
    self.assertEqual(len(exercises),1)
AssertionError: 2 != 1
```

**Comment by rtibbles** [comment]
There is no conceivable way that these two are related, but we should sort
it.

**Comment by benjaoming** [comment]
Yeah, I thought so, too.. so maybe it's a test with a race condition.

**Comment by rtibbles** [comment]
Seems an odd place for it. The other alternative, which seems more
plausible, is that it is not robust to changes in content packs.

**Comment by rtibbles** [comment]
Persistent, think it is related to a new content_pack.

```
======================================================================
FAIL: test_playlist_progress (kalite.coachreports.tests.coachreport_tests.PlaylistProgressResourceTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/ubuntu/ka-lite/kalite/coachreports/tests/coachreport_tests.py", line 141, in test_playlist_progress
    self.assertEqual(len(exercises),1)
AssertionError: 2 != 1

======================================================================
FAIL: test_topic_availability (kalite.updates.tests.availability_tests.TestTopicAvailability)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/ubuntu/ka-lite/kalite/updates/tests/availability_tests.py", line 23, in test_topic_availability
    self.assertEqual(topic["available"], any_available, "Topic availability for {topic} did not match child availability when any children are available.".format(topic=topic.get("title")))
AssertionError: False != True : Topic availability for Comparing two-digit numbers did not match child availability when any children are available.

======================================================================
FAIL: test_explore_overall (kalite.topic_tools.tests.explore_tests.TestExploreMethods)
get_explore_recommendations()
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/ubuntu/ka-lite/kalite/topic_tools/tests/explore_tests.py", line 63, in test_explore_overall
    self.assertEqual(actual[0].get("interest_topic").get("id"), "in-sixth-grade-math")
AssertionError: u'arithmetic' != 'in-sixth-grade-math'

----------------------------------------------------------------------
```

**Comment by rtibbles** [comment]
Have replicated two of the three test failures above locally, although curiously, I got `3 != 1` for the first failure.

**Comment by rtibbles** [comment]
Fixed in #4960
